lib/model/message_list.dart

```diff
@@ -381,6 +381,7 @@ class MessageListView with ChangeNotifier, _MessageSequence {
     for (final message in result.messages) {
       if (_messageVisible(message)) {
         _addMessage(message);
+        store.recentSenders.handleMessage(message);
```

Comment:
I am not sure whether to add it inside the if block or outside!!

Reply:
Good question.

What does the dartdoc on that `_messageVisible` method say?

Based on that, what would be the consequence of having that condition control whether to make this call? And what would be the consequence of ignoring that condition when making this call?